### PR TITLE
docs: Clarify what is meant by Architecture Group and what they do

### PR DESCRIPTION
**Pull request by sarina**
The Architecture Group is a backstop; they should help with Arbiter selection when requested, but it is not required to include them.

Clarify that currently the "Architecture Group" is the collection of members of the `#architecture` Slack channel.

### CODE REVIEW

**Review by robrap — APPROVED** [review]

File: oeps/processes/oep-0001.rst [hunk 1]
```diff
@@ -145,17 +145,21 @@ stability) to allow for additional incremental updates.
 Architecture Group
 ------------------
 
-The Architecture Group is ultimately accountable for assigning Arbiters to incoming
-Draft OEPs and revived OEPs that need a new Arbiter (if the original Arbiter is no
-longer available). The group can also be a resource to help or advise the
-Arbiter with the OEP process. The group can be found in the `Architecture Group
-Discourse category`_ or the ``#architecture`` channel in the `Open edX Slack`_.
+The Architecture Group serves as a backstop for the OEP process. Specifically,
```
[the remark this answers is not in the document]
Comment (sarina) [reply oeps/processes/oep-0001.rst:148]: Would it be worth adding here, and also adding in Step 1 Find an Arbiter (https://github.com/openedx/open-edx-proposals/blob/8ae5fb51c88cd2d9489fa5d03ded5aa07835b6cb/oeps/processes/oep-0001.rst#step-1-find-an-arbiter), a line like "It is encouraged, but not required, to find an Arbiter from a different organization than the Author." ? I don't feel that's too big of a change as it's just adding a recommendation, but would respect disagreement.

[the remark this answers is not in the document]
Comment (robrap) [reply oeps/processes/oep-0001.rst:148]: At edX we were happy with arbiters from different teams, but you may have new goals around this. I'll leave this to you to decide how important this is. Also, it is just a recommendation, so it would be fine by me.

[the remark this answers is not in the document]
Comment (sarina) [reply oeps/processes/oep-0001.rst:148]: That's fair. I might reword it to "It is encouraged, but not required, to find an Arbiter from a different group or organization than the Author" then. I think your point is fair.

[the remark this answers is not in the document]
Comment (robrap) [reply oeps/processes/oep-0001.rst:148]: That sounds good.

[the remark this answers is not in the document]
Comment (robrap) [reply oeps/processes/oep-0001.rst:148]: I also wish we could resolve the rst line break issue, either through linting to keep consistent, or an OEP-19 update stating either way is fine. I don’t like manually having to bring it up, knowing there is so much disagreement.

**Review by robrap — APPROVED** [review]
